Can't import

The example doesn't work:

```
const jisho = require('jisho.js')
// Error: Cannot find module 'jisho.js'
```

Obviously I have actually installed it and the dir does exist in node_modules.

What does work is:

```
const jisho = require('./node_modules/jisho.js/jisho')
```

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "jisho",
   "version": "1.0.0",
   "description": "Node wrapper module for the Japanese dictionary jisho.org.",
-  "main": "index.js",
+  "main": "jisho.js",
   "scripts": {
     "start": "node ."
   },

